test/fork/SNstCalls.t.sol

```diff
@@ -62,32 +62,32 @@ contract EthereumControllerSwapNSTToSNSTTests is ForkTestBase {
 
 contract EthereumControllerSwapSNSTToNSTFailureTests is ForkTestBase {
 
-    function test_swapSNSTToNST_notRelayer() external {
+    function test_withdrawFromSNST_notRelayer() external {
         vm.expectRevert(abi.encodeWithSignature(
             "AccessControlUnauthorizedAccount(address,bytes32)",
             address(this),
             RELAYER
         ));
-        ethereumController.swapSNSTToNST(1e18);
+        ethereumController.withdrawFromSNST(1e18);
     }
 
-    function test_swapSNSTToNST_frozen() external {
+    function test_withdrawFromSNST_frozen() external {
         vm.prank(freezer);
         ethereumController.freeze();
 
         vm.prank(relayer);
         vm.expectRevert("EthereumController/not-active");
-        ethereumController.swapSNSTToNST(1e18);
+        ethereumController.withdrawFromSNST(1e18);
     }
 
 }
 
 contract EthereumControllerSwapSNSTToNSTTests is ForkTestBase {
```

Comment:
Name should be changed to `EthereumControllerWithdrawFromSNSTTests `.

Reply:
Fixed